packages/client/src/jwt.ts

```diff
@@ -24,8 +24,13 @@ export const fetchJwks = async (url: string): Promise<JWK[]> => {
 };
 
 export const decode = (token: string) => {
-  const { payload } = jwt.decode(token, { complete: true });
-  const { iss, sub, aud, exp, iat, at_hash } = payload;
+  const jwtDecoded = jwt.decode(token, { complete: true });
```

Comment:
just wondering if jwt.decode throws any exceptions that need to be caught ? By default it returns falsy/empty value?

Reply:
No, it won't throw, just return null.